pkg/apis/extensions/types.go

```diff
@@ -933,6 +933,16 @@ type PodSecurityPolicySpec struct {
 	// privilege escalation.
 	// +optional
 	AllowPrivilegeEscalation bool
+	// AllowedHostPaths is a white list of allowed host paths. Empty indicates that all host paths may be used.
+	// +optional
+	AllowedHostPaths []AllowedHostPath
+}
+
+// AllowedHostPath defines the host volume conditions that will be enabled by a policy
+// for pods to use. It requires the path prefix to be defined.
+type AllowedHostPath struct {
+	// PathPrefix is the path prefix that the host volume must match
```

Comment:
Probably want to add an example here and talk about what "match" means. For better or worse these comments are largely our documentation about the APi. e.g. is there an difference between `/foo` and `/foo/`? Is `*` allowed?